test-alerts.cpp

```diff
@@ -1,8 +1,29 @@
 #define CATCH_CONFIG_MAIN  // This tells Catch to provide a main() - only do this in one cpp file
-
+#include <unistd.h>
 #include "test/catch.hpp"
 #include "typewise-alert.h"
 
 TEST_CASE("infers the breach according to limits") {
   REQUIRE(inferBreach(12, 20, 30) == TOO_LOW);
```

Comment:
Good set of tests for covering limit related paths... 👍 
Coverage missing for alert targets.. (or am I unable to locate those tests ?)..